libero-config/config--base.js

```diff
@@ -0,0 +1,55 @@
+const Color = require('color');
+
+const config = { data: {} };
+
+config.data.baselinegrid = {space: {}};
+config.data.baselinegrid.space.extra_small_in_px = 12;
+config.data.baselinegrid.space.small_in_px = config.data.baselinegrid.space.extra_small_in_px * 2;
+config.data.baselinegrid.space.smallish_in_px = config.data.baselinegrid.space.small_in_px * 1.5;
+config.data.baselinegrid.space.medium_in_px = config.data.baselinegrid.space.small_in_px * 2;
+config.data.baselinegrid.space.large_in_px = config.data.baselinegrid.space.small_in_px * 3;
+config.data.baselinegrid.space.extra_large_in_px = config.data.baselinegrid.space.small_in_px * 5;
+
+config.data.breakpoints = {site: {}};
+config.data.breakpoints.site.x_small = 320;
+config.data.breakpoints.site.small = 480;
+config.data.breakpoints.site.medium = 730;
+config.data.breakpoints.site.wide = 900;
+config.data.breakpoints.site.x_wide = 1200;
+
+config.data.color = { primary: {} };
+config.data.color.primary.normal = Color('#0288D1');
+config.data.color.primary.light = config.data.color.primary.normal.lighten(0.1);
```

Comment:
One small limitation of the current approach concerns 'relative' colours. For example, `primary.light` here is specified as a lighter shade of `primary.normal`. If `primary.normal` is overridden in a later config file, then `primary.lighten` would also have to be respecified, otherwise `primary.light` would still hold this old value.

Reply:
Will see if `jexl` might help here (/HT @thewilkybarkid for the suggestion).

Reply:
In node you could use getters:
```
$ node -i
> const config = { 
>    number: 1, 
>    get doubleNumber() { 
>        return this.number * 2; 
>    } 
> }

> config.number
1

> config.doubleNumber
2

> config.number = 2
2

> config.doubleNumber
4
```